dissect/target/plugins/os/windows/regf/shimcache.py

```diff
@@ -339,12 +343,12 @@ def shimcache(self) -> ShimcacheRecord:
                     self.target.log.warning("Error parsing ShimCache entry: %s %s", key, value_name)
                     continue
 
-                yield from self._get_records(cache)
+                yield from self._get_records(cache, value_name)
```

Comment:
```suggestion
                yield from self._get_records(value_name, cache)
```
Nitpicking, but you're passing the cache opened from `value_name`, so putting `value_name` first feels more natural.